src/inc/DefaultSettings.h

```diff
@@ -14,14 +14,12 @@ Author(s):
 --*/
 #pragma once
 
-constexpr COLORREF COLOR_WHITE = 0x00ffffff;
-constexpr COLORREF COLOR_BLACK = 0x00000000;
+constexpr til::color COLOR_WHITE{ 0xff, 0xff, 0xff };

```

Comment:
the 3-arg version of the til::color constructor sets alpha to 0xff. Alpha is only used if necessary. When converted to a COLORREF, it is ignored. Since we pushed colors/colorrefs all the way through, and retained the "with_alpha" calls inside TControl, it is safe to get rid of the WITH_ALPHA versions.